Gemfile

```diff
@@ -83,7 +82,7 @@ gem 'omniauth-twitter'
 gem 'omniauth-flickr', '>= 0.0.15'
 
 # client for Elasticsearch. Elasticsearch is a flexible
-# and powerful, distributed, real-time search and analytics engine. 
+# and powerful, distributed, real-time search and analytics engine.
```

Comment:
It's not a big deal, but you can make reviewers' lives easier by separating whitespace-only changes into separate commits.